ehitchhiking-ui/src/app/shared/components/blacklist/blacklist.component.sass

```diff
@@ -2,32 +2,27 @@
 @import '../../../../styles/fonts-constants'
 @import '../../../../styles/constants'
 
-.blacklist-modal-container
-  position: relative
-.blacklist-title
-  text-align: center
-.blacklist
-  height: 20rem
-  overflow-y: scroll
-  overflow-x: hidden
-  margin: $default-offset * 0.5
-  .blacklist-user
-    border: 1px solid $black
-    display: flex
-    justify-content: space-around
-    align-items: center
-    .profile-photo
-      font-size: $display-2
-    .remove-icon
-      font-size: $title
-      display: table
-      vertical-align: middle
-      color: $dark-red
-    .blacklist-user-name
-      font-size: $button
-      margin-top: $default-offset
-      width: 10rem
-.blacklist-back-btn
-  display: flex
-  justify-content: center
-
+.modal-wrapper
+  .modal-content
```

Comment:
height: 20rem
    overflow-y: scroll
    overflow-x: hidden
    padding: 0

move to modals.sass

Reply:
removed: overflow-y: scroll
overflow-x: hidden
padding: 0